keyboards/redox_w_cygnus/keymaps/default/keymap.c

```diff
@@ -0,0 +1,127 @@
+     KC_DEL          ,KC_1            ,KC_2            ,KC_3            ,KC_TRNS                                   ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,
+  //├────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤                         ├────────────────┼────────────────┴────────────────┼────────────────┤────────────────┤
+                                       KC_TRNS         ,KC_TRNS         ,KC_TRNS                                   ,KC_TRNS         ,KC_TRNS         ,KC_TRNS
+  //                                  └────────────────┴────────────────┴────────────────┘                         └────────────────┴────────────────┴────────────────┘
+  ),
+
+  [_ADJUST] = LAYOUT(
+  //┌────────────────┬────────────────┬────────────────┬────────────────┬────────────────┐                         ┌────────────────┬────────────────┬────────────────┬────────────────┬────────────────┐
+     KC_F1           ,KC_F2           ,KC_F3           ,KC_F4           ,KC_F5                                      ,KC_F6          ,KC_F7           ,KC_F8           ,KC_F9           ,KC_F10          ,
+  //├────────────────┼────────────────┼────────────────┼────────────────┼────────────────┐                         ┌────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+     KC_LGUI         ,KC_LALT         ,KC_LCTL         ,KC_LSFT         ,KC_TRNS                                   ,KC_F11          ,KC_F12          ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,
+  //├────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤                         ├────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+     KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,KC_TRNS                                   ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,KC_TRNS         ,
+  //├────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤                         ├────────────────┼────────────────┴────────────────┼────────────────┤────────────────┤
+                                       KC_TRNS         ,KC_TRNS         ,KC_TRNS                                   ,KC_TRNS         ,KC_TRNS         ,KC_TRNS
+  //                                  └────────────────┴────────────────┴────────────────┘                         └────────────────┴────────────────┴────────────────┘
+  )
+
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    switch (get_highest_layer(state)) {
+        case _QWERTY:
+            set_led_off;
+            break;
+        case _SYMB:
+            set_led_green;
+            break;
+        case _NAV:
+            set_led_blue;
+            break;
+        case _ADJUST:
+            set_led_red;
+            break;
+        default:
+            break;
```

Comment:
Something like this is preferable. Regardless, the appropriate functions should be directly called, rather than using a macro.
```suggestion
        case _SYMB:
            writePinLow(D1);
            break;
        case _NAV:
            writePinLow(F4);
            break;
        case _ADJUST:
            writePinLow(F5);
            break;
        default:
            writePinHigh(F5);
            writePinHigh(D1);
            writePinHigh(F4);
            writePinHigh(D0);
```